.profile.khan

```diff
@@ -5,6 +5,9 @@
 # is called for all sh-compatible shells.  So we put non-bashisms here
 # and bashisms in .bash_profile.
 
+# Add homebrew to path on M1 macs
+export PATH=/opt/homebrew/bin:$PATH
```

Comment:
Probably not necessary, but could we test for mac type before adding this? It won't cause any problem, but might be slightly confusing to some people why a dir they don't have locally is in their path.

To be clear, it's not a big deal, i'm just curious.

Reply:
It is essential on M1 macs.

It shouldn't hurt intel macs unless a boot disk from a M1 is moved to intel. I don't think we need to worry about this, but I'm only doing this on M1 now.

Reply:
Oh, no, I agree, sorry if I wasn't clear. I was saying it could be confusing for people, not the machine. 

I'm not sure if there's an easy way to test if a mac is an M1 or not; if so, we should consider only updating the path for the correct system. But it's not super important, so don't bother unless it takes like 2 minutes to implement.